tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -1,5 +1,3 @@
-# flake8: noqa
-data="""---
 namespace.tendrl.gluster_integration:
```

Comment:
change this to "namespace.gluster"

check https://github.com/Tendrl/ceph-integration/pull/165/files#r107025073